source/install/trouble_mysql.rst

```diff
@@ -0,0 +1,282 @@
+MySQL Installation Troubleshooting
```

Comment:
I started to review this section but it doesn't look clean yet due to indentation issues. Please use the "View file" option to preview the document and that way you can easily see what needs to be fixed. Please let me know any questions.

Also, new documentation pages should be added to the table of contents.

Reply:
The doc is starting to look better, but I suggest taking another look at this comment.

Reply:
Ya, I will fix and let you know.

Reply:
Looks better, however, the document is not yet added to the table of contents. To do this, you can go to https://docs.mattermost.com/guides/administrator.html and then add a link to the document in the correct place. Probably the "Installing Mattermost" section makes sense for this.